context/Cargo.toml

```diff
@@ -34,7 +33,7 @@ uuid = { version = "1.10.0", features = ["v5"] }
 wasm-bindgen = { version = "0.2.95", optional = true }
 magnus = { version = "0.7.1", optional = true, default-features = false }
 proto = { path = "../proto" }
-prost-wkt-types = "0.6.0"
+prost-wkt-types = { version = "0.5.1", features = ["vendored-protox"] }
```

Comment:
Until https://github.com/fdeantoni/prost-wkt/pull/70 lands, `prost-wkt-types` prevents us from having `prost`'s version be consistent which is required for generated code to compile properly